docker-compose.yml

```diff
@@ -10,7 +10,25 @@ services:
       - "./:/app"
     environment:
       - PYTHONUNBUFFERED=1
+      - FLASK_APP=iam/main.py
       - CONFIGURATION=dev
+      - SECRET_KEY=
+      - SERVICE_URL=
+      - FEAT_TOGGLE_LOCAL_AUTH=1
+      - FEAT_TOGGLE_FIREBASE=
+      - SQLALCHEMY_DATABASE_URI=postgres://postgres:@postgres:5432/iam
```

Comment:
If you add this as a default value for the variable, you don't have to overwrite this in the `Makefile` but docker compose would pick it up automatically from a `.env` file, for example.

Reply:
The override in Makefile is only there to specify the test database `iam_test` which is separate from the development database `iam`. So you would still need to override it when running tests, even if the app has a defined default.

Reply:
Fair point.